ids.json

```diff
@@ -2,11 +2,13 @@
 	"kimoServerID": "1193663232041304134",
     "kimoChannelID": "1193665461699739738",
     "kimoChannelDungeonID": "1202747652345962496",
+    "botLogChannelID":"1202553664539983882",
```

Comment:
All I did for ids.json is reorganize it so that the Server, Channels, and Roles are all grouped together.